mmcls/models/backbones/vision_transformer.py

```diff
@@ -9,7 +9,7 @@
 from mmengine.model import BaseModule, ModuleList
 from mmengine.model.weight_init import trunc_normal_
 
-from mmcls.registry import MODELS
+from ..builder import MODELS
```

Comment:
```suggestion
from mmcls.registry import MODELS
```
Please explain the reason of the modification?